staging/src/k8s.io/apiserver/pkg/apis/apiserver/v1alpha1/types.go

```diff
@@ -167,3 +167,54 @@ type TracingConfiguration struct {
 	// Defaults to 0.
 	SamplingRatePerMillion *int32 `json:"samplingRatePerMillion,omitempty" protobuf:"varint,2,opt,name=samplingRatePerMillion"`
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// PriorityAndFairnessConfiguration holds API Priority and Fairness parameters.
+type PriorityAndFairnessConfiguration struct {
+	metav1.TypeMeta `json:",inline"`
+
+	// WorkEstimatorConfiguration holds work estimator parameters.
+	// +optional
+	WorkEstimatorConfiguration WorkEstimatorConfiguration `json:"workEstimator,omitempty"`
+}
+
+// WorkEstimatorConfiguration holds work estimator parameters.
+type WorkEstimatorConfiguration struct {
+	// ListWorkEstimatorConfiguration holds work estimator parameters related to list requests.
+	// +optional
+	ListWorkEstimatorConfiguration ListWorkEstimatorConfiguration `json:"listRequests,omitempty"`
+	// MutatingWorkEstimatorConfiguration holds work estimator
+	// parameters related to watches of mutating objects.
```

Comment:
s/related to watches of mutating objects/related to requests that mutate watched objects/

Reply:
Changed.